src/helpers.js

```diff
@@ -1 +1,89 @@
+import axios from 'axios'
+
 export const toUnixTimestamp = d => d ? new Date(d).getTime() / 1000 : undefined
+
+
+/**
+ * Map frontend column names to API expected values
+ */
+export const sortByMapping = {
+  'messages': 'MESSAGES',
+  'posts': 'POST',
+  'aggregates': 'AGGREGATE',
+  'store': 'STORE',
+  'program': 'PROGRAM',
+  'instance': 'INSTANCE',
+  'forget': 'FORGET'
+}
+
+/**
+ * Fetch addresses with pagination, sorting and filtering
+ * Only uses v1 API endpoint
+ * 
+ * @param {Object} api_server The API server configuration
+ * @param {Object} options Query parameters
+ * @returns {Promise<Object>} The API response
+ */
+export const fetchAddresses = async (api_server, options = {}) => {
+  const {
+    page = 1,
+    perPage = 20,
+    sortBy = 'messages',
```

Comment:
`'messages'` is not accepted by the api.

Change default fallback to `sortByMapping.messages` to keep single source of truth and avoid this kind of issues


```suggestion
    sortBy = sortByMapping.messages,
```